### PR TITLE
Update service.markdown

### PR DESCRIPTION
**Pull request by Trev-W**
Include example of how to publish a tasmota command - took me 2 hours of trial and error to find out that the publish and subscribe topic structures are different and also to work out the correct format, this might save someone else some anguish.

## Proposed change
 Include example of how to publish a tasmota command - took me 2 hours of trial and error to find out that the publish and subscribe topic structures are different and also to work out the correct format, this might save someone else some anguish.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/mqtt/service.markdown [hunk 1]
```diff
@@ -48,6 +48,13 @@ qos: 2
 retain: true
 ```
 
+Example of how to publish a Tasmota command (LedPower 2):
+
+```yaml
+topic: your_tasmota_device_name_topic/cmnd/LEDPOWER
+payload: 2
+```
```
Comment (frenck) [inline source/_docs/mqtt/service.markdown:56]: This is a duplicate of the very first example (with different values).

Why add duplicates?

**Comment by Trev-W** [comment]
Because not everyone is a skilled programmer such as yourself.  I "wasted"
2 hours by trial and error trying to solve this exact problem. Had there
been more examples I would've been able to see a pattern and arrive at the
solution quicker, so I decided to add my amendment so that someone else in
the future may be assisted.

What is the cost of having multiple examples? Where is the detriment?

**Comment by frenck** [comment]
Well, the "detriment" here, it that we, in that case, could add 50 examples, but if the exact one triggering your brain doesn't exist, it will still be a problem.

For that specific reason, I don't see a need for adding this example, since it is a duplicate.

Please note, this documentation is not here to learn people about MQTT itself.

**Comment by Trev-W** [comment]
There is a problem with the documentation in this project being too sparse/
terse, with not enough examples. This raised it's head in the forums around
the time of the WTH month, and those who posted voicing this opinion were
quickly told well if you think there's a problem with the documentation
make an edit.

This was my first attempt at a documentation edit, next time I'll ensure
that my contribution is entirely novel👍

**Comment by frenck** [comment]
Sure, but this is not a documentation problem, like the example you are adding already exists. This is a lack of knowledge of MQTT. We do not provide documentation for how MQTT itself works, we provide documentation around the MQTT integration.

Learning people MQTT itself is out of scope for the project.

Nevertheless, I see why you want to add this, but going to friendly decline this addition.

**Pull request closed without merging**